content/Teams/observability-platform/sre.md

```diff
@@ -0,0 +1,119 @@
+# Observability Platform Team SRE Processes
+
+This document explains a few processes related to operating our production software together with AppSRE team. All operations can be summarised as the Site Reliability Engineering part of our work.
+
+## Goals
+
+The main goals of our SRE work within this team is to deploy and operate software that meets functional (API) and performance (SLO) requirements of our customers.
```

Comment:
Link to the SLO document you started?

Reply:
Also - link to the dashboards that include our SLOs at the moment? `Telemeter` one I think.

Reply:
Added proper sections in our docs in handbook (even if TBD)